roles/pulp3-devel/tasks/main.yml

```diff
@@ -8,29 +8,22 @@
           - dnf-utils
           - dstat
           - fd-find
-          - fedora-easy-karma
-          - fpaste
           - fzf
-          - git
           - htop
           - httpie
           - iotop
           - jnettop
           - jq
           - ncdu
           - python-django-bash-completion
-          - python-ipdb
-          - python-rpdb
           - python3-setuptools
           - python3-virtualenvwrapper
           - redhat-lsb-core
           - ripgrep
           - screen
-          - telnet
           - tmux
           - tree
           - wget
-          - vim-enhanced
```

Comment:
I use vim-enhanced every session. I do not use the others that are removed so those are OK w/ me.

Reply:
I suggested we remove, but leaving it in is fine by me.